MnAu2/mnau2_fm.scf.in

```diff
@@ -0,0 +1,32 @@
+ &control
+    calculation = 'scf',
+    pseudo_dir = '../pseudo',
+    outdir='../temp'
+    prefix='mnau2',
+ /
+ &system
+    ibrav=  0, 
+    nat=  3, 
+    ntyp= 3,
+    ecutwfc = 60.0, ecutrho = 800.0,
+    occupations='smearing', smearing='mv', degauss=0.02,
+    nspin=2, 
+    starting_magnetization(1)= 0.8,
+ /
+ &electrons
+    mixing_beta = 0.3 
+ /
+CELL_PARAMETERS
```

Comment:
The cell parameters from the structure file that I emailed to you (which these numbers are from) are in units of angstroms, not atomic units (units of Bohr radii). Quantum Espresso can use those units, provided you add `angstrom` to the right of `CELL_PARAMETERS`, yielding

```
CELL_PARAMETERS angstrom
 3.3701291084         0.0000000000         0.0000000000
 0.0000000000         3.3701291084         0.0000000000
-1.6850646988        -1.6850646988         4.3794687067
```